apps/avifenc.c

```diff
@@ -36,18 +36,19 @@ typedef struct
     avifBool qualityAlphaIsConstrained; // true if qualityAlpha explicitly set by the user
     int overrideQuality;
     int overrideQualityAlpha;
-    avifBool progressive;
+    avifBool autoProgressive;
     int speed;
+    int layers;
     avifHeaderFormat headerFormat;
 
-    int paspCount;
-    uint32_t paspValues[8]; // only the first two are used
-    int clapCount;
+    avifBool paspValid;
```

Comment:
`paspValid==AVIF_FALSE` implies `--pasp` was defined with incorrect values to me, not that it is unspecified.  
What about `overridePasp` for consistency with `overrideQuality`? Or `paspPresent` , `paspSet` maybe?

Reply:
Just found if user specified `--crop` and then `--clap` avifenc will silently treat first 4 clap values as crop values and convert them. This should be fixed in another PR.